packages/lwt/lwt.2.5.1/opam

```diff
@@ -40,5 +40,6 @@ depopts: [
 conflicts: [
  "react" {<"1.0.0"}
  "ssl" {< "0.5.0"}
+ "camlp4" {= "4.02+7"} (* lacks "camlp4.quotations.o" *)
```

Comment:
interesting... did you discuss this with upstream lwt?  as mentioned, it works fine here (camlp4-4.02+7 and lwt-2.5.1)